flutter/lib/src/sentry_flutter.dart

```diff
@@ -90,6 +91,9 @@ mixin SentryFlutter {
 
     await _initDefaultValues(flutterOptions, channel);
 
+    flutterOptions
+        .addPerformanceCollector(SpanFrameMetricsCollector(flutterOptions));
```

Comment:
should this go inside `_initDefaultValues`? so we are consistent with other stuff

Reply:
yeh makes sense